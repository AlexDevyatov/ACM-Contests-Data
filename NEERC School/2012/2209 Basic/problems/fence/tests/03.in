8
-25 0
-24 -16
-21 -22
13 -25
28 -6
28 20
25 22
-10 25
-12 4
