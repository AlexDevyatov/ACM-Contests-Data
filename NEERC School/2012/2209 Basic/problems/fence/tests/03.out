9
-25 0
-24 -16
-21 -22
13 -25
28 -6
28 20
-12 4
25 22
-10 25
